.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,11 +3,19 @@ Thank you for submitting a PR to pylint!
 
 To ease the process of reviewing your PR, do make sure to complete the following boxes.
 
-- [ ] Write a good description on what the PR does.
-- [ ] Create a news fragment with `towncrier create <IssueNumber>.<type>` which will be
-  included in the changelog. `<type>` can be one of: breaking, user_action, feature,
-  new_check, removed_check, extension, false_positive, false_negative, bugfix, other, internal.
-  If necessary you can write details or offer examples on how the new change is supposed to work.
+- [ ] Document your change, if it is a non-trivial one.
+  - A maintainer might label the issue ``skip-news`` if the change does not need to be in the changelog.
+  - Otherwise, create a news fragment with ``towncrier create <IssueNumber>.<type>`` which will be
+    included in the changelog. ``<type>`` can be one of: breaking, user_action, feature,
+    new_check, removed_check, extension, false_positive, false_negative, bugfix, other, internal.
```

Comment:
```suggestion
    included in the changelog. ``<type>`` can be one of the types defined in `./towncrier.toml`.
```

Reply:
Ha right. But this isn't going to change a lot, maybe it's better if we have it directly here ?

Reply:
Just adds one more thing that can get out of sync. This is mostly for new people anyway so it doesn't hurt if they get pointed to where stuff actually gets defined rather than a `.MD` file.